app/views/home/index.html.erb

```diff
@@ -1,26 +1,23 @@
 <div class='card-deck'>
-   <div class='card'>
-      <div class='card-block'>
-         <h4 class='card-title'>iPhone 5S</h4>
-         <h6 class='card-subtitle text-muted'>Rp. 8.999.999</h6>
+  <% @books.each do |book|%>
```

Comment:
for readability, please insert a single space after the last bar "|" before the "%"